src/mtginator/game.py

```diff
@@ -144,16 +169,20 @@ def mulligan(self, rules=None, n=7, verbose=True):
 
     def enumerate_plays(self):
         ''' For a given hand (or metahand) and available mana, what plays are available to Player
-             Returns:  Set of Card objects
+             Returns:  List of Card objects
```

Comment:
I think I made this a Set() on purpose, like what if you have Forest, Forest, Forest, Llnaowar Elves?   You don't need an array of 4.    But possibly i overlooked something.